.github/workflows/continuous-integration.yml

```
@@ -87,7 +59,7 @@ jobs:
       - name: setup and run yamllint on .yaml and .md files
         run: |
           pip install yamllint
```

I think as we only use it in Python scripts we should put it into `scripts/`
```suggestion
          pip install -r scripts/requirements.txt
```